ui/apps/platform/src/types/databaseService.proto.ts

```diff
@@ -0,0 +1,5 @@
+export type DatabaseStatus = {
+    databaseAvailable: boolean;
+    databaseType?: string | null;
+    databaseVersion?: string | null;
```

Comment:
Although `?` optional is needed, `| null` is not also needed based on reply from David Shrewsberry in https://github.com/stackrox/stackrox/pull/4369/files#r1068572917

> I don't think you will receive a null unless your (java|type)script interprets an empty string as such. Here is what a curl will provide you:

```js
{"databaseAvailable":false,"databaseType":"Hidden","databaseVersion":""}
```

notice the databaseVersion is an empty string.